www_src/lib/panzoom-jquery.js

```diff
@@ -0,0 +1,1199 @@
+/* jshint ignore:start */
```

Comment:
Not sure if it does exactly what we need, but the matrix utils in [math.js](https://github.com/josdejong/mathjs) are really nice.